Lib/pdb.py

```diff
@@ -2077,34 +2075,44 @@ def help():
 
 
 def main():
-    import getopt
-
-    opts, args = getopt.getopt(sys.argv[1:], 'mhc:', ['help', 'command='])
-
-    if not args:
-        print(_usage)
+    import argparse
+
+    parser = argparse.ArgumentParser(prog="pdb",
+                                     description=_usage,
+                                     formatter_class=argparse.RawDescriptionHelpFormatter,
+                                     allow_abbrev=False)
+
+    parser.add_argument('-c', '--command', action='append', default=[], metavar='command')
+    group = parser.add_mutually_exclusive_group(required=True)
+    group.add_argument('-m', metavar='module')
```

Comment:
I suggest using names longer than 1 letter, you should update the code below as well.

```suggestion
    group.add_argument('-m', metavar='module', dest='module')
```